docs/design/autoware-architecture/planning/index.md

```diff
@@ -1,46 +1,104 @@
 
 ![overall-planning-architecture](image/high-level-planning-diagram.drawio.svg)
 
-The Planning component consists of the following sub-components:
+Following the principles of microautonomy architecture, we have adopted a modular system framework. The functions within the Planning domain are implemented as modules, dynamically or statically adaptable according to specific use cases. This includes modules for lane changes, intersection handling, and pedestrian crossings, among others.
+
+The Planning Component comprises several sub-components:
+
+- **Mission Planning**: This module calculates routes from the current location to the destination, utilizing map data. Its functionality is similar to that of Fleet Management Systems (FMS) or car navigation route planning.
+- **Planning Modules**: These modules plans the vehicle's behavior for the assigned mission, including target trajectory, blinker signaling, etc. They are divided into Behavior and Motion categories:
+  - **Behavior**: Focuses on calculating safe and rule-compliant routes, managing decisions for lane changes, intersection entries, and stoppings at a stop line.
+  - **Motion**: Works in cooperate with Behavior modules to determine the vehicle's trajectory, considering its motion and ride comfort. It includes lateral and longitudinal planning for route shaping and speed calculation.
+- **Validation**: Ensures the safety and appropriateness of the planned trajectories, with capabilities for emergency response. In cases where a planned trajectory is unsuitable, it triggers emergency protocols or generates alternative paths.
+
+### Highlights
+
+Key aspects of this high-level design include:
+
+#### Modulation of each function
+
+Essential Planning functions, such as route generation, lane changes, and intersection management, are modularized. These modules come with standardized interfaces, enabling easy addition or modification. More details on these interfaces will be discussed in subsequent sections.
+
+#### Separation of Mission Planning sub-component
+
+Mission Planning serves as a substitute for functions typically found in existing services like FMS (Fleet Management System). Adherence to defined interfaces in the high-level design facilitates easy integration with third-party services.
+
+#### Separation of Validation sub-component
+
+Given the extendable nature of the Planning Component, ensuring consistent safety levels across all functions is challenging. Therefore, the Validation function is managed independently from the core planning modules, maintaining a baseline of safety even for arbitrary changes of the planning modules.
+
+#### Interface for HMI (Human Machine Interface)
+
+The HMI is designed in high-level for smooth cooperation with human operators. These interfaces enable coordination between the Planning Component and operators, whether in-vehicle or remote.
```

Comment:
I don't think `in high-level` is correct.
```suggestion
The HMI is designed for smooth cooperation with human operators. These interfaces enable coordination between the Planning Component and operators, whether in-vehicle or remote.
```
Otherwise Bard suggests the following:
```
The HMI is designed with a high-level abstraction to facilitate intuitive and efficient interaction with human operators.
```